terraform/sns.tf

```diff
@@ -7,3 +7,8 @@ resource "aws_sns_topic" "yara_match_alerts" {
 resource "aws_sns_topic" "metric_alarms" {
   name = "${var.name_prefix}_binaryalert_metric_alarms"
 }
+
+//No YARA match alerts will be published to this SNS topic.
+resource "aws_sns_topic" "safe_alerts" {
+  name = "${var.name_prefix}_binaryalert_safe_alerts"
+}
```

Comment:
Instead of making this required for all BinaryAlert deployments, let's make it a configurable option:

1. Add a new config variable to `terraform.tfvars` which is `enable_safe_alerts`
2. Create this SNS topic only when needed:

```hcl
resource "aws_sns_topic" "safe_alerts" {
    count = "${var.enable_safe_alerts}"
    name = ...
}
```
3. The analyzer checks whether the safe SNS topic is defined, skipping it if it is not